src/styles/catalog_card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__content {
+    margin-top: 40px;
+  }
+
+  &__title {
+    color: $main-text-color;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__code {
+    margin-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__rate {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__review {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 16px;
+    color: $main-text-color;
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-text {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $secondary-color;
+    }
+
+    &-cost {
+      font-style: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $main-text-color;
+    }
+  }
+
+  &__link {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 16px;
+    width: 100%;
+    border-radius: 5px;
+    height: $ling-height;
+    background-color: $blue-color;
+    border: 1px solid $blue-color;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-decoration: none;
+    cursor: pointer;
+    transition-duration: $animationTransitionDuration;
+
+    &:hover {
+      background-color: #fff;
+      color: $blue-color;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__title {
+      color: #34568b;
+    }
+  }
```

Comment:
Let's add a transition for this hover effect too.
You can use sass shorthand nesting here as well for .card__title.